app/controllers/show-project.js

```diff
@@ -10,6 +10,11 @@ export default class ShowProjectController extends Controller {
   @argument shareClosed = true;
   @argument copySuccess = false;
 
+  flagText = '';
+  flagClosed = true;
+  flagSuccess = false;
+  reCaptchaResponse = null;
```

Comment:
In general, unless these things depend on query params (which are inherently Controller level properties), stuff like this should be kept inside a component, including the `bblFeatureCollectionLayer`.

Reply:
Done